playbooks/roles/openvpn/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Import the public GPG key from OpenVPN community
+  apt_key: url=https://swupdate.openvpn.net/repos/repo-public.gpg state=present
+
+- name: Adds the OpenVPN community repositroy to sources.list
+  apt_repository: repo='deb http://swupdate.openvpn.net/apt trusty main' state=present
```

Comment:
The convention is for each argument to go on a separate line, so `state=present` would be on its own line and indented to match `repo=...`. The other tasks in this file follow the same format.

This file hasn't been converted to multi-line YAML syntax yet (which makes this even more explicit) but it will be soon.